examples/40_advanced/example_text_preprocessing.py

```diff
@@ -0,0 +1,76 @@
+# -*- encoding: utf-8 -*-
+"""
+==================
+Text Preprocessing
+==================
+This example shows, how to use text features in *auto-sklearn*. *auto-sklearn*
+can automatically encode text features if they are provided as string type.
+
+For processing text features you need a pandas dataframe and set the desired
+text columns to string and the categorical columns to category.
+
+*auto-sklearn* ass text embedding creates a bag of words count
```

Comment:
ass?

Reply:
Perhaps it should be put in bold, i.e. **ass**

Reply:
.. but more likely just removed